### PR TITLE
Add Slowlogs command

### PR DESCRIPTION
**Pull request by luiztiago**
## Description

Customers could use the CLI to list Slow Query Logs as a similar feature used on the VIP Dashboard.

## Steps to Test

1. Check out PR.
1. Run `npm run build` and `npm link`
1. Run `node ./dist/bin/vip @SITE_ID.production slowlogs`
1. Verify list of slowlogs or "No logs found" if don't have any slowlogs to be listed

### CODE REVIEW

File: src/bin/vip-slowlogs.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+		org_id: opt.app.organization.id,
+		app_id: opt.app.id,
+		env_id: opt.env.id,
+		limit: opt.limit,
+		follow: opt.follow || false,
+		format: opt.format,
+	};
+}
+
+function printSlowlogs( slowlogs, format ) {
```
Comment (pschoffer) [inline src/bin/vip-slowlogs.js]: There is a build-in formatting feature to vip-cli already. Could we use that instead to stay consistent? Or is your format too different?

Example of usage:
https://github.com/Automattic/vip-cli/blob/update/sync_error_handling/src/bin/vip-config-envvar-get-all.js#L77

**Review by pschoffer — DISMISSED** [review]
This looks great! The only thing that comes to mind is resuing existing formatting logic

File: src/bin/vip-slowlogs.js (hunk 1 above)
Comment (luiztiago) [reply src/bin/vip-slowlogs.js]: @pschoffer Good catch, I'll test it with the build-in format. It should works well :)

Comment (luiztiago) [reply src/bin/vip-slowlogs.js]: @pschoffer done, thanks :)

File: src/bin/vip-slowlogs.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+	// Strip out __typename
+	slowlogs = slowlogs.map( log => {
+		const { timestamp, rowsSent, rowsExamined, queryTime, requestUri, query } = log;
+
+		return { timestamp, rowsSent, rowsExamined, queryTime, requestUri, query };
+	} );
+
+	// let output = '';
```
Comment (rinatkhaziev) [inline src/bin/vip-slowlogs.js]: Should this block be removed?

File: src/bin/vip-slowlogs.js
Comment (rinatkhaziev) [inline src/bin/vip-slowlogs.js:1]: Probably should implement in TS from the start? as there's ongoing work to convert everything to TS and it's likely we'll migrate to something like oclif in the near future.

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
Thanks! left a bit of inline feedback

**Comment by luiztiago** [comment]
@rinatkhaziev Thanks, updates are done. Good catches! :)

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**